diff --git a/src/node-commands.ts b/src/node-commands.ts
index e7d8b50..9534892 100644
--- a/src/node-commands.ts
+++ b/src/node-commands.ts
@@ -538,7 +538,7 @@ export function registerNodeCommands(program: Command) {
 
         const txDetails = {
           from,
-          to: '0x0000000000000000000000000000000000000001',
+          to: '0x0000000000000000000000000000000000010000',
           gasPrice,
           gasLimit: 30000000,
           value,
@@ -616,7 +616,7 @@ export function registerNodeCommands(program: Command) {
 
         const txDetails = {
           from,
-          to: '0x0000000000000000000000000000000000000001',
+          to: '0x0000000000000000000000000000000000010000',
           gasPrice,
           gasLimit: 30000000,
           data: ethers.utils.hexlify(
